js/app.jsx

```diff
@@ -297,101 +416,94 @@ var CaseverListItem = React.createClass({
   }
 });
 
+var SearchTableHeadConstTh = React.createClass({
+  render: function(){
+    return(
+      <th>{this.props.text}</th>
+    )
+  }
+})
 
-var CaseverList = React.createClass({
+var SearchTableHeadTh = React.createClass({
+  handleSort: function(){
+    var newOrder = "";
+    if (this.state.order == "") {
+      newOrder = "-";
+    }
+    this.setState({"order": newOrder})
+    this.props.handleAddFilter(' orderby:' + newOrder + this.props.filter, / orderby:[-\w]+/)
+  },
+  getInitialState: function(){
```

Comment:
Can we move this state to `SearchableList`? If we keep the state in each `<th/>`, it can't know whether its sibling `<th/>`'s are sorting or not.